extensions/interactions/ItemSelectionInput/static/item_selection_input.css

```diff
@@ -76,16 +71,26 @@ md-checkbox.item-selection-input-checkbox .md-label {
   width: 8px;
 }
 
-.multiple-choice-option:hover .multiple-choice-outer-adio-button {
+.multiple-choice-option:focus .multiple-choice-outer-radio-button {
   border-color: #115FD4;
 }
 
-.multiple-choice-outer-radio-button {
-  border: 2px solid #0D48A1;
-  border-radius: 50%;
-  display: inline-block;
-  height: 16px;
-  margin-bottom: -1px;
-  margin-right: 12px;
-  width: 16px;
+/* Hover pseudo-class should be used in desktops only. */
```

Comment:
Why?

Reply:
Hover is not relevant in mobile devices, I think. Also, we are going to use a button for multiple choice input now.

Reply:
If it is just not relevant do we need special case for it? If it breaks something on mobile it is understandable but the comment should mention that.

Reply:
I don't understand what you mean by "If it is just not relevant do we need special case for it?", sorry. 

The "selection-on-hover" functionality will remain only for desktop, which is why I moved hover related stuff into the media query section. It will not be active in mobile screens.

Reply:
I mean if this change breaks something on mobile then it makes sense to have it separated but the comment should specify that it is separated because it breaks stuff on mobile.

Reply:
OK, I'll change the comment. Thanks.

Reply:
Done.